test/http-test.js

```diff
@@ -267,8 +267,108 @@ describe('HTTP', function() {
       isvalid: true,
       address: addr.toString(node.network),
       scriptPubKey: Script.fromAddress(addr).toRaw().toString('hex'),
-      ismine: false,
-      iswatchonly: false
+      iswitness: false,
+      isscript: false
+    });
+  });
+
+  it('should not validate invalid address', async () => {
+    // send an address for the wrong network
+    const json = await nclient.execute('validateaddress', [
+      addr.toString('main')
+    ]);
+    assert.deepStrictEqual(json, {
+      isvalid: false
+    });
+  });
+
+  it('should validate a p2wpkh address', async () => {
+    const info = await wallet.createAccount('foo3', {
```

Comment:
Necessary to create an account here, or is this to get an address? I doesn't really need the wallet, right? It could be better to use a vector for a p2wpkh.